tests/test_playbooks/fixtures/redhat_manifest-0.yml

```diff
@@ -45,7 +45,7 @@ interactions:
     status: {code: 200, message: OK}
 - request:
     body: !!python/unicode '{"facts": {"distributor_version": "sat-6.3", "system.certificate_version":
-      "3.2"}, "type": "satellite", "name": "katello.example.com"}'
+      "3.2"}, "contentAccessMode":"entitlement", "type": "satellite", "name": "katello.example.com"}'
```

Comment:
Yes, I "manually" edited that, mostly because re-recording this is a pain. (manually = re-record but only commit that one hunk).